streaming/src/test/scala/org/apache/spark/streaming/StreamingListenerSuite.scala

```diff
@@ -386,3 +400,14 @@ class StreamingContextStoppingCollector(val ssc: StreamingContext) extends Strea
     }
   }
 }
+
+class CustomizedStreamingListener extends StreamingListener {
```

Comment:
Shall we use a different word instead of `Customized`?

Reply:
Done